src/EnergyPlus/Furnaces.cc

```diff
@@ -9935,6 +9950,74 @@ namespace Furnaces {
 
 	}
 
+	void
+	SetMinOATCompressor(
+		int const FurnaceNum, // index to furnace
+		std::string const FurnaceName, // name of furnace
+		std::string const cCurrentModuleObject, // type of furnace
+		std::string const CoolingCoilType, // type of cooling coil
+		std::string const CoolingCoilName, // name of cooling coil
+		std::string const HeatingCoilType, // type of heating coil
+		std::string const HeatingCoilName, // name of heating coil
```

Comment:
Yeah, I'm concerned about the performance hit due to this passing strings around.  If this was just being called once, it wouldn't be a big deal, but (and correct me if I'm wrong) it looks like this is the main `sim` interface, so this will be called lots of times.  If there is some reason strings should be passed here, then I think we are going to want to pass them via const reference  (`const std::string & CoolingCoilName`) to avoid making string copies.